sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -173,6 +187,7 @@ using pi_command_list_map_t =
     std::unordered_map<ze_command_list_handle_t, pi_command_list_info_t>;
 // The iterator pointing to a specific command-list in use.
 using pi_command_list_ptr_t = pi_command_list_map_t::iterator;
+// The type that holds a command list and the queue descriptor.
```

Comment:
```suggestion
```

Reply:
I don't see any difference between what's there and the suggested change.

Reply:
remove this line

Reply:
Yes, done.